chromadb/db/migrations.py

```diff
@@ -58,6 +58,11 @@ def __init__(self, path: str, db_hash: str, source_hash: str):
         )
 
 
+class InvalidHashError(Exception):
```

Comment:
"InconsistentHashError" needs to be updated to not be md5 specific

Reply:
Good catch, updated :+1: